core/scheduler/src/main/kotlin/Repeating.kt

```diff
@@ -33,7 +48,7 @@ sealed interface Repeating {
     data class Daily(
         override val hour: Int = 0,
         override val minute: Int = 0,
-        override val second: Int = 0
+        override val second: Int = 0,
```

Comment:
Not sure how I feel about trailing commas, but that's a separate issue

Reply:
https://recology.info/2019/02/trailing-commas/

Dangling commas make patches clearer to read and reduce slightly conflicts, or make them easier